### PR TITLE
Add new action T093 - precursor chemical licenses

### PR DESCRIPTION
**Pull request by huwd**
relates to: https://trello.com/c/0PlCII8Q/303-t093-new-action-drug-precursor-chemicals-home-office

- Adds: new action
- Test visible
  - here: https://finder-frontend-pr-1621.herokuapp.com/get-ready-brexit-check/results?c%5B%5D=chemical&c%5B%5D=import-from-eu&c%5B%5D=export-to-eu&c%5B%5D=owns-operates-business-organisation, and
  - here: https://finder-frontend-pr-1621.herokuapp.com/get-ready-brexit-check/results?c%5B%5D=non-metal-material&c%5B%5D=import-from-eu&c%5B%5D=owns-operates-business-organisation

### CODE REVIEW

File: lib/brexit_checker/change_notes.yaml [hunk 1]
```diff
@@ -42,3 +42,7 @@ change_notes:
      action_id: T064
      note: "Link changed to provide hauliers with updated guidance."
      date: 2019-10-02
+   - uuid: "7683bdd6-eef4-4d78-b78f-9e507e0ba0da"
+     type: addition
+     action_id: T093
+     date: 2019-10-03
```
Comment (benthorner) [inline lib/brexit_checker/change_notes.yaml]: Spotted afterwards: we should update this to today's date.

Comment (huwd) [reply lib/brexit_checker/change_notes.yaml]: on it, will have to change this on all these batch of notifications! Whoops!
Thanks for spotting.

Comment (huwd) [reply lib/brexit_checker/change_notes.yaml]: done

File: lib/brexit_checker/change_notes.yaml [hunk 2]
```diff
@@ -42,3 +42,7 @@ change_notes:
      action_id: T064
      note: "Link changed to provide hauliers with updated guidance."
      date: 2019-10-02
+   - uuid: "7683bdd6-eef4-4d78-b78f-9e507e0ba0da"
+     type: addition
+     action_id: T093
+     date: 2019-10-07
```
Comment (huwd) [inline lib/brexit_checker/change_notes.yaml]: Changed this date in hopes we can get this up today

**Pull request merged**